charts/kube-prometheus-stack/Chart.yaml

```
@@ -23,8 +23,8 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 34.9.0
-appVersion: 0.55.0
+version: 35.2.1
```

New feature should raise minor version.
```suggestion
version: 35.3.0
```

---

done